modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -215,6 +249,28 @@ private CompletableFuture<?> loadSnapshotMeta(ClusterNode snapshotSender) {
         }
     }
 
+    private boolean metadataIsSufficientlyComplete() {
+        return partitionSnapshotStorage.catalogService().latestCatalogVersion() >= snapshotMeta.requiredCatalogVersion();
+    }
+
+    private void logMetadataIsInsufficiencyAndSetError() {
+        LOG.warn(
+                "Metadata not yet available, URI '{}', required level {}; rejecting snapshot installation.",
```

Comment:
Is this the way we format logs? I thought it should be more like "Foo [a=1, b=2]", there a guidelines somewhere